cli/cmd/secrets.go

```diff
@@ -73,8 +95,6 @@ func createSecret() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.Username, "username", "u", "", "username")
-	cmd.Flags().StringVarP(&opts.Password, "password", "p", "", "password")
 	cmd.Flags().StringVarP(&opts.Description, "description", "d", "", "Secret description")
```

Comment:
Still not sure about having a `--description` flag, unless there's an important use-case that we're addressing